largescale/config/many-collection-testing

```diff
@@ -1,5 +1,17 @@
+enable_stats_check=true
 insert_rate=0
 limit_throughput=100000
+max_avg_commands_latency=80000
```

Comment:
I will like a separate section for the thresholds, different from other config parameters.

```
# Config parameters
....
# Thresholds if configured
....
```

Would that mean we will have to add the ability to ignore comments

Reply:
4b7f3805bd190e7dc6bef16e3e32ea722ee8e875